static/components/search.js

```diff
@@ -0,0 +1,53 @@
+const search = instantsearch({
+  appId: process.env.ALGOLIA_APP_ID,
+  apiKey: process.env.ALGOLIA_ADMIN_KEY,
+  indexName: "apprenticeships",
+  routing: true,
+  searchableAttributes: ["company", "description", "location"],
+  searchFunction: function(helper) {
+    helper.search();
+  }
+});
+
+search.addWidget(
+  instantsearch.widgets.hits({
+    container: "#hits",
+    hitsPerPage: 10,
+    templates: {
+      allItems: `
```

Comment:
Is 'esversion: 6' not es6? Any reason we aren't using the latest version of JS? I recognize that we would need a transpile step with Babel, but isn't that okay since we are using Parcel? Thoughts? I can rewrite to use es5 standards.